lib/commands/getValue.js

```diff
@@ -34,12 +34,12 @@ let getValue = function (selector) {
             throw new CommandError(7, selector || this.lastResult.selector)
         }
 
-        let elementIdAttributeCommands = []
+        let elementIdPropertyCommands = []
         for (let elem of res.value) {
-            elementIdAttributeCommands.push(this.elementIdAttribute(elem.ELEMENT, 'value'))
```

Comment:
Can we have a fallback here that (like [here](https://github.com/webdriverio/webdriverio/blob/master/lib/protocol/keys.js#L48)) to ensure that this still works for driver that haven't implemented the `elementIdProperty` command?